arviz/data/io_cmdstan.py

```diff
@@ -746,6 +747,9 @@ def _process_configuration(comments):
             elif match_float:
                 key, value = match_float.group(1), match_float.group(2)
                 results[key] = float(value)
+            elif match_str_bool:
+                key, value = match_str_bool.group(1), match_str_bool(2)
```

Comment:
match_str_bool(2) I believe should be matr_str_bool.group(2)

Reply:
Additionally, because the results dictionary values are converted to strings the value `False` is read incorrectly as `"False"` when attempting to convert to an integer on line 807 of this PR `int(pconf.get("save_warmup", 0))`

Reply:
Good catch, let's just turn these to int